.evergreen/connectivity-tests/Dockerfile

```
@@ -1,5 +1,5 @@
 # "bullseye" is the debian distribution that ubuntu:20.04 is based on
-FROM node:18-bullseye
+FROM node:20-bullseye
```

Sumer small suggestion so that we don't need to bump this manually in the future. We can use docker ARG to make the FROM statement dynamic. Will need to update the Dockerfile here:

```suggestion
ARG NODE_JS_VERSION=20-bullseye
FROM node:${NODE_JS_VERSION}
```

And the command in .evergreen/connectivity-tests/run.sh:

```diff
- docker build -t devtools-connectivity-tests -f "./.evergreen/connectivity-tests/Dockerfile" .
+ docker build -t devtools-connectivity-tests --build-arg "NODE_JS_VERSION=$NODE_JS_VERSION" -f "./.evergreen/connectivity-tests/Dockerfile" 
```

---

Sure – done in 6384e143591cc84447f41cdf57556e1fbb31c48c!